ui/src/flux/components/DatabaseListItem.tsx

```diff
@@ -1,14 +1,21 @@
 import React, {PureComponent, ChangeEvent, MouseEvent} from 'react'
 import classnames from 'classnames'
+import {CopyToClipboard} from 'react-copy-to-clipboard'
 
 import {tagKeys as fetchTagKeys} from 'src/shared/apis/flux/metaQueries'
 import parseValuesColumn from 'src/shared/parsing/flux/values'
 import TagList from 'src/flux/components/TagList'
-import {Service} from 'src/types'
+import {
+  notifyCopyToClipboardSuccess,
+  notifyCopyToClipboardFailed,
+} from 'src/shared/copy/notifications'
+
+import {Service, Notification} from 'src/types'
 
 interface Props {
   db: string
   service: Service
+  notify: (message: Notification) => void
```

Comment:
somewhat out of scope for this PR, but could this be a NotifyAction type that could be used throughout the codebase?

Reply:
i have created the action and changed the notify types in the components in pr using it.